### ISSUE TITLE
Update to 1.20.4

### ISSUE THREAD

**Issue by mrjasonn** [JorelAli/CommandAPI#510]
### Description

Please do would be very useful as there are other plugins waiting on you to update

**Comment by JorelAli** [comment]
Thanks for the issue! We are aware that 1.20.4 has been released. We typically try to get the CommandAPI up-to-date within a week of Minecraft's release, which would put us at Tuesday 12th December for 1.20.3, or Friday 15th December for 1.20.4.

With the update to 1.20.3 (and 1.20.4), there were some considerable changes internally with regards to how functions return success values and result values, which has made figuring out how to update the corresponding `FunctionArgument` very challenging. It may be that we will have to (temporarily?) drop support for this argument in order to make a timely release.

We were expecting to get out a release by Wednesday 6th December, but unfortunately I caught covid and have been unable to make progress on updating the CommandAPI at this time due to being in bed for the past week.

We do have snapshot builds of 9.3.0, which can be accessed via our Maven snapshot repository which _may_ (as of https://github.com/JorelAli/CommandAPI/commit/34d44cb7dcc2f3af9cd7a7bd9eeb26c29361984a) be compatible with 1.20.4. Instructions for accessing the snapshot repository via Maven or Gradle are currently on our Discord (I'll write instructions on GitHub on how to do this when I can). Direct downloads for plugin snapshot versions can be found here: https://s01.oss.sonatype.org/content/repositories/snapshots/dev/jorel/commandapi-bukkit-plugin/. These are to be used at your own discretion (they've not been thoroughly tested like releases and may have bugs in them).

Last time I looked, there were a few outstanding bugs that were holding back 1.20.4's release.
